examples/tests/valid/expressions_binary_operators.w

```diff
@@ -6,3 +6,5 @@ let xyz = ((y * y) / (x * x)) * z;
 let xf = 1.0;
 let yf = (-20.22) * xf;
 let zf = (xf + (yf) - -0.01);
+let xyzf = 501 \ (99 + 1);
+assert(xyzf == 5);
```

Comment:
I'm assuming we're aiming for integer division here. So lets also add a test for negative numbers which will show that using `Math.floor(..)` is a bug here.
```suggestion
let xyzf = 501 \ (99 + 1);
assert(xyzf == 5);
let xyznf = -501 \ (99 + 1);
assert(xyznf == -5);
```